src/components/Header/announcement.tsx

```diff
@@ -35,7 +35,7 @@ const Announcement = () => {
     if (isProduction && (isHome || isPortal)) {
       return (
         <>
-          Scroll {requireEnv("REACT_APP_SCROLL_ENVIRONMENT")} is now live. <ReadMoreLink href="/portal">Try it!</ReadMoreLink>
+          Scroll {requireEnv("REACT_APP_SCROLL_ENVIRONMENT")} is now live. <ReadMoreLink>Try it!</ReadMoreLink>
```

Comment:
"Try it" will no longer direct users on the Home page to the Portal page, maybe we can just display the announcement on the Home page (not on the Portal page anymore)?

Reply:
Make sense,  modified the tag type, this entire line can be clicked to move to the portal~